src/algorithms/calorimetry/CalorimeterHitReco.cc

```diff
@@ -276,6 +277,9 @@ void CalorimeterHitReco::process(
             sid,
             lid,
             local_position);
+#if EDM4EIC_VERSION_MAJOR >= 7
+        recohit.setRawHit(rh);
+#endif
```

Comment:
@veprbl Do we care about unused variables when taking false `#if` branch? `[[maybe_unused]]`?

Reply:
done